library/spdm_requester_lib/libspdm_req_get_capabilities.c

```diff
@@ -85,6 +91,11 @@ static bool validate_responder_capability(uint32_t capabilities_flag, uint8_t ve
                 (hbeat_cap == 1) || (key_upd_cap == 1)) {
                 return false;
             }
+            if (version == SPDM_MESSAGE_VERSION_13) {
```

Comment:
I'd move this down to line 156 with the rest of the 1.3 checks.

Reply:
I add it here purposely.

event_cap relies on other session cap. Most logic can be merged.

Reply:
This breaks the flow of the checking with respect to SPDM versions. Here, 1.3 capabilities are introduced before 1.2 capabilities.

Reply:
I think we need to use the most efficient way to do the check.

Otherwise, if we use check 1.0, check 1.1, check 1.2, check 1.3 style, there will be lots of duplication.